lib/cartopy/mpl/gridliner.py

```diff
@@ -38,6 +38,14 @@
     cartopy.crs.SouthPolarStereo,
     cartopy.crs.Stereographic
 )
+_ROTATE_LABEL_PROJS = _POLAR_PROJS + (
```

Comment:
No that's not the opposite and the list is quite subjective. This a controversial point that we must clarify.

Reply:
That is fine for it to be subjective I think. As long as a user can control on/off rotation later I think this is OK to at least get a reasonable default list together. I was just wondering if there was an easier way to build it based off of the _crs.py file instead of making a list here.